pkg/acr/projectapi.go

```diff
@@ -54,7 +54,7 @@ func (p *projectAPI) GetByName(name string) (globalregistry.Project, error) {
 			return project, nil
 		}
 	}
-	return nil, nil
+	return nil, fmt.Errorf("no project found to add members: %w", globalregistry.ErrRecoverableError)
```

Comment:
Why are we returning an error here? (nil,nil) is usually the "not found" return value. What has the members to do with the `GetByName` method?

Reply:
In ACR without an actual repo, we don't have projects at all. We can have some logic in memberAddAction.Perform, but I thought it is better to have an error here, than creating it for different providers in the reconcile package.

The Error message is wrong, fixed!